src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -129,6 +129,15 @@
       <SourcesToCopyToTestDir Include="@(OverridenRuntimeContent)" />
       <DestinationsForTestDir Include="@(OverridenRuntimeContent->'$(TestPath)\%(Filename)%(Extension)')" />
     </ItemGroup>
+    
+    <!-- Removing assets from TestCopyLocal that come in from P2P references in SourcesToCopyToTestDir -->
+    <ItemGroup>
+        <_FileNamesInSourcesToCopyToTestDir Include="@(SourcesToCopyToTestDir->'%(Filename)%(Extension)')">
```

Comment:
nit: To match the convention this should also be named _SourceToCopyToTestDirByFileName